Specify minimal required Rust version

The pkg-config 0.3.10 update broke downstream crates on Rust 1.21...

```
error[E0599]: no method named `to_ascii_uppercase` found for type `char` in the current scope
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/pkg-config-0.3.10/src/lib.rs:538:28
    |
538 |     name.chars().map(|c| c.to_ascii_uppercase()).map(|c| {
    |                            ^^^^^^^^^^^^^^^^^^
    |
    = help: items from traits can only be used if the trait is in scope
    = note: the following trait is implemented but not in scope, perhaps add a `use` for it:
            candidate #1: `use std::ascii::AsciiExt;`
```

I run cargo-audit in CI for most of my crates, which depends on pkg-config (directly or indirectly). Unfortunately this means that I cannot use cargo-audit in CI for crates supporting Rust 1.21 without adding a separate build step that runs on a newer version.

Would it be possible to specify the minimal required Rust version, to add that version to CI and to consider changes to the minimal version a breaking change?

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,5 +1,6 @@
 language: rust
 rust:
+  - 1.13.0
   - stable
   - beta
   - nightly

--- a/README.md
+++ b/README.md
@@ -1,6 +1,7 @@
 # pkg-config-rs
 
 [![Build Status](https://travis-ci.org/alexcrichton/pkg-config-rs.svg?branch=master)](https://travis-ci.org/alexcrichton/pkg-config-rs)
+[![Rust](https://img.shields.io/badge/rust-1.13%2B-blue.svg?maxAge=3600)](https://github.com/alexcrichton/pkg-config-rs/)
 
 [Documentation](https://docs.rs/pkg-config)
 
@@ -11,6 +12,8 @@ library is located.
 You can use this crate directly to probe for specific libraries, or use
 [metadeps](https://github.com/joshtriplett/metadeps) to declare all your
 `pkg-config` dependencies in `Cargo.toml`.
+
+This library requires Rust 1.13+.
 
 # Example
 

